src/tabs/tabs.js

```diff
@@ -75,17 +121,41 @@
         this.CssClasses_.MDL_JS_RIPPLE_EFFECT_IGNORE_EVENTS);
     }
 
-    // Select element tabs, document panels
+    // Select element tab_bar, tabs, document panels
     this.tabs_ = this.element_.querySelectorAll('.' + this.CssClasses_.TAB_CLASS);
     this.panels_ =
         this.element_.querySelectorAll('.' + this.CssClasses_.PANEL_CLASS);
+    this.tabsBar_ =
+        this.element_.querySelector('.'  + this.CssClasses_.TAB_BAR_CLASS);
 
     // Create new tabs for each tab element
     for (var i = 0; i < this.tabs_.length; i++) {
       new MaterialTab(this.tabs_[i], this);
     }
 
     this.element_.classList.add(this.CssClasses_.UPGRADED_CLASS);
+
+    this.tabsBar_.setAttribute('role', 'tablist');
```

Comment:
Same with the other roles. Isn't this up to the developer? They have markup access.